app/experimenter/experiments/models/nimbus.py

```diff
@@ -172,6 +172,19 @@ def treatment_branches(self):
             branches = branches.exclude(id=self.reference_branch.id)
         return list(branches)
 
+    @property
```

Comment:
I might be over-thinking this. But, I figured we'd want some more logic here:

* Only report a timeout if the experiment has gotten kicked from WAITING back to REVIEW - and is *still* in REVIEW publishing status.
* The timeout reported is None if/when the experiment gets another attempt going from REVIEW -> APPROVED -> WAITING